packages/flutter/lib/src/widgets/image.dart

```diff
@@ -142,7 +142,8 @@ class Image extends StatefulWidget {
     this.matchTextDirection: false,
     this.gaplessPlayback: false,
     this.package,
-  }) : image = new NetworkImage(src, scale: scale),
+    Map<String, String> headers,
```

Comment:
the docs for this constructor should mention the headers argument too.

Reply:
I did it or I don't understand, I added a test however @Hixie